frontend/src/App.svelte

```diff
@@ -108,6 +135,16 @@
       >
     </p>
 
+    {#if genEdQuery}
+      <div class="flex text-sm mb-2">
```

Comment:
Did you try it on a mobile-sized screen? I thought we needed to flex-wrap

Reply:
Rather than having it wrap around, this will shrink the front line -- this is what it looks like in Mobile S:

<img width="472" alt="Screenshot 2023-07-22 at 2 39 56 PM" src="https://github.com/ekzhang/classes.wtf/assets/39931478/421e5a89-5513-416b-9ec3-5ecba36ba7d4">

Reply:
I like this a lot more than the original plan to wrap the checkboxes! I can try to rationalize about why this looks nicer to me: In general, the only wrapping we do right now is on blocks of text, which wrap naturally and people expect to wrap. If we were to wrap an area of interaction like checkboxes, it might feel unnatural.

Reply:
However, there is a lot of empty space. One alternative that I'd like to see (which is also a little spacey) is reversing the strategy: Keep the text unchanged but tighten the checkboxes, perhaps into a 2x2 grid.

Reply:
this looks fine to me, Cole's ideas sound good too

Reply:
keeping it as is! Thank you all for the feedback